useraccount/internal/ports/drivenframework.go

```diff
@@ -1 +1,10 @@
 package ports
+
+type PostgresSQLPort interface {
+	OpenDBConnection()
+	CloseDBConnection()
+	AddNewPersonInfo()
+	DeletePersonInfo()
```

Comment:
Not going to implement

Reply:
Not sure I understand the comment

Reply:
We are not going to implement delete CRUD